herddb-core/src/main/java/herddb/cluster/BookkeeperCommitLog.java

```diff
@@ -616,7 +616,7 @@ public void recovery(
                                     lastLedgerId = ledgerId;
                                     currentLedgerId = ledgerId;
                                     lastSequenceNumber.set(entryId);
-                                    if (number.after(snapshotSequenceNumber)) {
+                                    if (number.after(snapshotSequenceNumber) && statusEdit.type != LogEntryType.TABLE_CONSISTENCY_CHECK) {
```

Comment:
This is not the right place.
I think it should be done in TableSpaceManager 
this class should not be aware of the semantics of entries